src/main/java/com/epam/grid/engine/provider/job/slurm/SlurmJobProvider.java

```diff
@@ -178,7 +178,7 @@ private void validateJobOptions(final JobOptions options) {
         if (!StringUtils.hasText(options.getCommand())) {
             throw new GridEngineException(HttpStatus.BAD_REQUEST, "Command should be specified!");
         }
-        if (options.getPriority() < 0) {
+        if (options.getPriority() != null && options.getPriority() < 1L) {
             throw new IllegalArgumentException("Priority should be between 0 and 4_294_967_294");
```

Comment:
we check that priority `< 1` but warn that it should be greater then `0`

Reply:
fixed